src/test/java/com/dsrm/dsrmbackend/AbstractIntegrationTest.java

```diff
@@ -25,7 +25,9 @@ public void initialize(ConfigurableApplicationContext applicationContext) {
                     "spring.datasource.username=" + container.getUsername(),
                     "spring.datasource.password=" + container.getPassword(),
                     "spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true",
-                    "spring.jpa.hibernate.ddl-auto=update"
+                    "spring.jpa.hibernate.ddl-auto=update",
+                    "dsrm.auth.jwtExpirationMs=3600000",
```

Comment:
we do not need that here. put this in application-test.properties in test->resources directory